Helper/Search/Search.php

```diff
@@ -53,7 +55,9 @@ public function __construct(ClientRequest $clientRequest)
         $this->types = $options['types']; //required
         $this->facets = $options['facets'] ?? [];
         $this->sizes = $options['sizes'] ?? [];
-        $this->setSorts(($options['sorts'] ?? []), $clientRequest->getLocale());
+        $this->defaultSorts = $this->parseSorts(($options['default_sorts'] ?? []), $clientRequest->getLocale());
```

Comment:
Does the default value should not be ['_score'] instead of []? I'm worry that a query with an empty array may won't work?

Reply:
If the default sort is a empty array the queryBuilder will remove the sort because it uses array_filter

Reply:
public function buildBody(): array
    {
        $postFilter = $this->getPostFilters();
        $hasPostFilter = $postFilter != null;
        return array_filter([
            'query' => $this->getQuery(),
            'post_filter' => $postFilter,
            'aggs' => $this->getAggs($hasPostFilter),
            'suggest' => $this->getSuggest(),
            'sort' => $this->getSort(),
            'highlight' => $this->search->getHighlight()
        ]);
    }